### PR TITLE
3.0.3 Patch Notes

### PR DESCRIPTION
**Pull request by ktollivervmw**
Updated release notes with update 3.0.3 release information.

@Margarita-Staneva 
Hi Margarita, due to the CVE last week we released a hot fix to address it. This commit is to update the release notes with the changes applied to Wavefront Nozzle for TAS 3.0.3.

### CODE REVIEW

File: docs-content/release-notes.html.md.erb [hunk 1]
```diff
@@ -3,10 +3,16 @@ title: Release Notes for VMware Tanzu Observability by Wavefront Nozzle
...
-<strong>Page last updated: November 18, 2021</strong>
+<strong>Page last updated: December 10, 2021</strong>
 
 These are release notes for VMware Tanzu Observability by Wavefront Nozzle.
 
+##<a id="3-0-3"></a> v3.0.3
+
+**Release Date:** December 10, 2021
+
+* Security fix for a vulnerability
```
Comment (Margarita-Staneva) [inline docs-content/release-notes.html.md.erb]: Should be 

* Security fix for a vulnerability.

File: docs-content/release-notes.html.md.erb [hunk 2]
```diff
@@ -3,10 +3,16 @@ title: Release Notes for VMware Tanzu Observability by Wavefront Nozzle
 owner: Partners
 ---
 
-<strong>Page last updated: November 18, 2021</strong>
+<strong>Page last updated: December 10, 2021</strong>
```
Comment (Margarita-Staneva) [inline docs-content/release-notes.html.md.erb]: Last updated date is the date on which the document is edited. Let's put the date on which the doc will be pushed out.

**Review by Margarita-Staneva — CHANGES_REQUESTED** [review]
I have 2 comments.

It will be great if we start the process for updating the docs before a release.

File: docs-content/release-notes.html.md.erb (hunk 1 above)
Comment (ktollivervmw) [reply docs-content/release-notes.html.md.erb]: Text has been updated.

File: docs-content/release-notes.html.md.erb (hunk 2 above)
Comment (ktollivervmw) [reply docs-content/release-notes.html.md.erb]: Updated to "December 14, 2021" since the release note needed to be updated today.

**Pull request merged**